tests/runner/origin/base_host.py

```
@@ -440,10 +456,11 @@ async def run_genvm(
 	host: str,
 	extra_args: list[str] = [],
 	# default config fee buckets use bucket_no 0, 1 and 2
-	bucket_totals: list[int] = [10_000_000, 10_000_000, 10_000_000],
+	bucket_totals: list[int],
 	code: bytes | None = None,
 	calldata: bytes,
 	leader_nondet_results: list[bytes] | None = None,
+	message_fee_allocation: list[MessageFeeAllocationNode] = [],
```

Ruff (0.15.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)